test/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -75,7 +75,9 @@ class CheckYourAnswersControllerSpec extends SpecBase with MockitoSugar with Sca
       ))
 
       val subscriptions: Seq[AnswerSection] = {
-        ua.get(SummarySubscriptionsPage)(PSubsByYear.pSubsByYearFormats).get.zipWithIndex.flatMap {
+        ua.get(SummarySubscriptionsPage)(PSubsByYear.pSubsByYearFormats).get.toSeq.sortBy{
+          case (taxYear, _) => -taxYear
```

Comment:
You don't need to define a sorting function. Use `NpsDataFormats.sort` which complies with the implementation as well.

Reply:
Ah, okay. I'll change it to use that